src/mame/snk/hng64_3d.ipp

```diff
@@ -5,14 +5,28 @@
 /// Hyper NeoGeo 64 - 3D bits ///
 /////////////////////////////////
 
+#define LOG_3D           (1 << 1)
+#define LOG_FRAMEBUFFER  (1 << 2)
+#define LOG_DISPLAY_LIST (1 << 3)
+#define LOG_TEXTURE      (1 << 4)
+
+#define VERBOSE (0)
+
+#include "logmacro.h"
+
+#define LOG3D(...)           LOGMASKED(LOG_3D, __VA_ARGS__)
+#define LOGFRAMEBUFFER(...)  LOGMASKED(LOG_FRAMEBUFFER, __VA_ARGS__)
+#define LOGDISPLAYLIST(...)  LOGMASKED(LOG_DISPLAY_LIST, __VA_ARGS__)
+#define LOGTEXTURE(...)      LOGMASKED(LOG_TEXTURE, __VA_ARGS__)
```

Comment:
This isn’t really safe to do in something that’s going to get #included somewhere else.  Does this even need to be a .ipp file?  Could it just be another .cpp file to implement the renderer classes?